aspnetcore/fundamentals/minimal-apis/overview.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Minimal APIs overview
+author: JeremyLikness
+description: An introduction to the fastest and easiest way to create web API endpoints with ASP.NET Core.
+ms.author: jeliknes
+monikerRange: '>= aspnetcore-6.0'
+ms.date: 11/22/2022
+uid: fundamentals/minimal-apis/overview
+---
+
+# Minimal APIs overview
+
+Minimal APIs are both a technology and an approach to quickly and easily create web-based APIs with HTTP endpoints in ASP.NET Core.
```

Comment:
Can we rephrase this to : "Minimal APIs are a ASPNET Core micro-framework for quickly and easily creating HTTP APIs with .NET" 

or "Minimal APIs are a simplified approach for building fast HTTP APIs with ASPNET Core". 

A few things:
- Can we avoid "Web-based"? 
- Can we make sure the "Minimalistic approach" and "perf=fast" show up in the wording ?

Reply:
> Can we rephrase this to : "Minimal APIs are a ASPNET Core micro-framework for quickly and easily creating HTTP APIs with .NET"

@JeremyLikness  I'll suggest a change with this and other changes.